mosaic-runtime/src/jvmMain/kotlin/com/jakewharton/mosaic/platform.kt

```diff
@@ -4,7 +4,7 @@ import java.nio.CharBuffer
 import java.nio.charset.StandardCharsets.UTF_8
 import org.fusesource.jansi.AnsiConsole
 
-private val out = AnsiConsole.out()!!
+private val out = AnsiConsole.out()!!.also { AnsiConsole.systemInstall() }
```

Comment:
This is not possible, because it's a top-level property:
```suggestion
AnsiConsole.systemInstall()
private val out = AnsiConsole.out()!!
```
and the order doesn't matter according to what `AnsiConsole` does.

alternative:
```suggestion
private val out = run { AnsiConsole.systemInstall(); AnsiConsole.out()!! }
```
but I think the also better expresses what is happening.